ansible/inventory/ec2.py

```diff
@@ -0,0 +1,1701 @@
+#!/usr/bin/env python
```

Comment:
Add reference to the source commit sha (or version tag) as comment in this file.

Reply:
added